server/routes/plan-summary/PlanSummaryController.ts

```diff
@@ -42,10 +42,11 @@ export default class PlanSummaryController {
     try {
       const planUuid = req.services.sessionService.getPlanUUID()
       const goals = await this.goalService.getGoals(planUuid)
```

Comment:
if this method was accessed via POST couldn't the correct list of goals be sent in the body, saving us this lookup and the selection from the map on line 46?

Reply:
No, because its used from tabs as well which is GET so dont want to handle things differently at 2 different places for same thing